### PR TITLE
[Merged by Bors] - feat(AlgebraicTopology/Quasicategory): alternative statement of hornfilling condition

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicTopology/Quasicategory.lean [hunk 1]
```diff
@@ -55,4 +55,25 @@ lemma Quasicategory.hornFilling {S : SSet} [Quasicategory S] ⦃n : ℕ⦄ ⦃i
 instance (S : SSet) [KanComplex S] : Quasicategory S where
   hornFilling' _ _ σ₀ _ _ := KanComplex.hornFilling σ₀
 
+open SimplicialObject SimplexCategory in
+lemma quasicategory_of_filler (S : SSet)
```
Comment (joelriou) [inline Mathlib/AlgebraicTopology/Quasicategory.lean:59]: Is it possible to extract from this proof the extensionality property that a morphism from a horn is determined by its restriction to suitable faces?

**Review by joelriou — APPROVED** [review]

File: Mathlib/AlgebraicTopology/Quasicategory.lean (hunk 1 above)
Comment (jcommelin) [reply Mathlib/AlgebraicTopology/Quasicategory.lean:58]: Good suggestion. Done.

**Comment by jcommelin** [comment]
I completely agree that we should fix universes. My hope / gut feeling is that it should be easy to do orthogonal to my series of PRs that show that the nerve of a category.{0} is a quasicategory.

**Comment by joelriou** [comment]
For the universe issue, I have started #9631 which is not too messy...

Thanks for this!

bors merge

**Pull request closed without merging**